MAUI/Popup/localization.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Localization in .NET MAUI SfPopup control | Syncfusion
+description: Learn here all about Localization support in Syncfusion .NET MAUI Popup (SfPopup) control and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Localization in .NET MAUI Popup (SfPopup)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfPopup` can be localized by adding `resource` file. 
```

Comment:
Localization is the process of translating the application resources into different languages for specific cultures. The `SfPopup` can be localized by adding a `resource` file.